apps/test/unit/code-studio/components/libraries/LibraryCreationDialogTest.js

```diff
@@ -82,56 +82,74 @@ describe('LibraryCreationDialog', () => {
   });
 
   describe('onOpen', () => {
+    let getJSLintAnnotationsStub, sourceStub, functionStub;
+    let libraryName = 'name';
+    let source = 'function foo() {}';
+    beforeEach(() => {
+      getJSLintAnnotationsStub = sinon.stub(
```

Comment:
nit: I'm not sure if there's a better name, but `getJSLintAnnotationsStub` to me reads like it's a function to get a stub for `JSLintAnnotations`

Reply:
I tried finding a better name, but couldn't come up with anything (stubGetJSLintAnnotations has the same problem). I'm going to leave it as-is.